lambda/tfe-backup/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/chanzuckerberg/go-misc/aws"
+	"github.com/chanzuckerberg/go-misc/lambda/tfe-backup/runner"
+	"github.com/getsentry/sentry-go"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/pkg/errors"
+)
+
+func run(ctx context.Context) error {
+	config := &runner.Config{}
+	envconfig.MustProcess("", config)
+
+	flushSentry, err := runner.SetupSentry(config.SentryDSN, config.SentryEnv)
+	if err != nil {
+		return err
+	}
+	defer flushSentry()
+
+	err = run0(ctx, config)
```

Comment:
Is this a Go idiom? This kinda looks to me like we want a private method, but doing so is difficult in Go. In JavaScript land I've seen closures used to do this and failing that conventions like `_function_name` or `__function_name` to signal that the method is not meant to be called from outside an object/context.

Reply:
not really, these probably could use better names. I was copy/pasting the structure from the other TFE lambda in the repo for consistency for now